tests/topotests/ospf6_ecmp_inter_area/r4/ospf6d.conf

```diff
@@ -0,0 +1,14 @@
+interface r4-eth0
+ ipv6 ospf6 hello-interval 2
+ ipv6 ospf6 dead-interval 10
+!
+interface r4-eth1
+ ipv6 ospf6 hello-interval 2
+ ipv6 ospf6 dead-interval 10
+!
+router ospf6
+ ospf6 router-id 10.254.254.4
+ redistribute connected
+ interface r4-eth0 area 0.0.0.0
```

Comment:
Suggested to use interface level area configuration for ospfv3.

Reply:
Done